guide/service/listen-for-tasks.md

```diff
@@ -142,12 +142,28 @@ Consider listening for tasks when your service is ready. If your service needs t
 <tab title="Node" vp-markdown>
 
 ```javascript
-const MESG = require('mesg-js').service()
-
-MESG.listenTask({
-// task      inputs           outputs
-  taskX: ({ inputX, inputY }, { outputX, outputY }) => outputX({ foo: "super result", bar: true })
+const mesg = require('mesg-js').service()
+
+mesg.listenTask({
+  __TASK_1_KEY__: (inputs, outputs) => {
```

Comment:
keep it just one task and call it `taskX` that's the stuff we have in the documentation (or should have) and the default on the js template